tracer/test/test-applications/integrations/Samples.Avro/sample.avsc

```diff
@@ -0,0 +1,48 @@
+{
+  "doc": "IF THE AVSC FILE IS CHANGED, SOURCES NEED TO BE REGENERATED\n To do so, install avro tools with\n `dotnet tool install Apache.Avro.Tools --create-manifest-if-needed`\n and then generate the files by running\n `dotnet avrogen -s tracer/test/test-applications/integrations/Samples.Avro/sample.avsc tracer/test/test-applications/integrations/Samples.Avro/ --skip-directories`",
```

Comment:
this is the only way to add comments in this file 🤷
the best would be to have a build action to regenerate the files, but doing so in a cross-platform way is a bit challenging.